### DISCUSSION TITLE
Better ranking algo

### DISCUSSION THREAD

**Discussion by aleksasiriski** [hearchco/agent#406] [Ideas]
Currently if a one engine returns it's first rank for a single result that isn't returned by any other engine, if the weights are the same across all engines, than that sole result will be at the top whereas results that get returned by multiple engines go way down because of the current algo (avering the rank across engines).

I believe that we should consider results that get returned by multiple engines better ones than the results that get returned by a single engine, by how much I'm not sure. This needs to be discussed.

Example: Mojeek returns some garbage as first result, none other return that same URL but it's considered by us the best result because it's score is 100, whereas a result that is ranked 1 by Google and 2 by Bing has a score less than 100 and is going to be placed lower in the list.

Would love your opinions on this @k4lizen @matijakljajic

**Comment by k4lizen** [comment]
Yeah sure, we can probably achieve / test that by changing the rconf.TRA variable to something larger than 1.
